infra/charts/feast/charts/feast-jc/values.yaml

```diff
@@ -0,0 +1,154 @@
+# replicaCount -- Number of pods that will be created
+replicaCount: 1
```

Comment:
We can run multiple of these side by side without a problem?

Reply:
not really. state is in memory, so only one pod per configuration (deployment)